platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/iss/IssDetector.java

```diff
@@ -232,13 +282,21 @@ public void signalEndOfPreconsensusReplay(@Nullable final Object ignored) {
     /**
      * Handle postconsensus state signatures.
      *
-     * @param transactions the signature transactions to handle
-     * @return a list of ISS notifications, or null if no ISS occurred
+     * @param round the round that may contain state signatures
+     * @return a list of ISS notifications, which may be empty
      */
-    public @Nullable List<IssNotification> handlePostconsensusSignatures(
-            @NonNull final List<ScopedSystemTransaction<StateSignatureTransaction>> transactions) {
-        return listOrNull(
-                transactions.stream().map(this::handlePostconsensusSignature).toList());
+    private @NonNull List<IssNotification> handlePostconsensusSignatures(@NonNull final ConsensusRound round) {
+        final List<ScopedSystemTransaction<StateSignatureTransaction>> stateSignatureTransactions =
+                new SystemTransactionExtractor<>(StateSignatureTransaction.class).handleRound(round);
```

Comment:
SystemTransactionExtractor should just be a static function, instantiating a new object shouldn't be needed.

Reply:
will be fixed in https://github.com/hashgraph/hedera-services/pull/11962